### PR TITLE
[seurat-scripts] Adds mscorefonts to avoid plots with no letters

### PR DESCRIPTION
**Pull request by pcm32**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Currently the mulled container for seurat-scripts produces plots with squares instead of letters. Based on https://github.com/conda-forge/r-base-feedstock/issues/91 this adds the font to alleviate the problem.

### CODE REVIEW

File: recipes/seurat-scripts/meta.yaml [hunk 1]
```diff
@@ -21,6 +20,7 @@ requirements:
         - r-seurat 2.3.1
         - r-optparse
         - r-workflowscriptscommon
+        - mscorefonts
```
Comment (dpryan79) [inline recipes/seurat-scripts/meta.yaml]: This will hopefully soon be taken care of by https://github.com/conda-forge/r-base-feedstock/pull/104

**Comment by dpryan79** [comment]
@bgruening Do you have any thoughts on this? I'm tempted to merge given the recent comment on the proposed r-base PR that pertains to this.

**Comment by dpryan79** [comment]
The newest iteration of handling this: https://github.com/conda-forge/r-feedstock/pull/15

Note that that would result in us needing to specify `r` rather than `r-base` in recipes.

**Comment by jdblischak** [comment]
> Note that that would result in us needing to specify r rather than r-base in recipes.

You may be better off manually specifying the fonts as a dependency. Add `r` will add `r-recommended`, which adds a bunch of [extra dependencies](https://github.com/conda-forge/r-recommended-feedstock/blob/master/recipe/meta.yaml#L15).

**Comment by dpryan79** [comment]
@jdblischak Good point

**Comment by pcm32** [comment]
Thanks for review, merge and comments @dpryan79 @jdblischak

**Comment by pcm32** [comment]
@dpryan79 apparently adding the mscorefonts as mentioned in the above issue didn't fix the problem of squares instead of letters with R plots:

image

Any other ideas I could use to fix this?

**Comment by pcm32** [comment]
...wait... I think Galaxy might have used the wrong container...

**Pull request merged**